final_task/rss_reader/parser_rss.py

```diff
@@ -0,0 +1,131 @@
+import json
+import re
+import signal
+import logging
+from contextlib import contextmanager
+from urllib.error import URLError
+import feedparser
+from dateutil import parser
+import News
+import html
+from exceptions import *
```

Comment:
It is better to not use wildcard import at all

Reply:
Done